src/preact/index.js

```diff
@@ -166,3 +167,12 @@ export function useImperativeHandle(ref, create, opt_deps) {
 export function toChildArray(unusedChildren) {
   return preact.toChildArray.apply(undefined, arguments);
 }
+
+/**
+ * @param {T} current
+ * @return {{current: T}}
+ * @template T
+ */
+export function useValueRef(current) {
```

Comment:
This should be moved within `src/preact/component`. This `src/preact/index` is reserved only for the standard Preact APIs.

Reply:
Thanks, updated in the latest!